staging/src/k8s.io/code-generator/cmd/validation-gen/validators/openapi.go

```diff
@@ -72,8 +72,11 @@ func (openAPIDeclarativeValidator) ExtractValidations(t *types.Type, comments []
 			return result, fmt.Errorf("multiple %s tags found", formatTagName)
 		}
 		format := formats[0]
-		formatFunction := FormatValidationFunction(format)
-		if formatFunction != nil {
+		if formatFunction, err := getFormatValidationFunction(format); err != nil {
+			return result, err
+		} else if formatFunction != nil {
```

Comment:
Shouldn't it be `formatFunction == nil`?

Reply:
Clearly so, dangit :)  Will send a new one

Reply:
again, bitten by lack of tests.